sos/plugins/elastic.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2018 Amit Ghadge <amitg.b14@gmail.com>
+
```

Comment:
You need to include the standard license header here: it is no longer the full GPL2 preamble, but a simple block of text that refers readers to the main LICENSE file:

```
# This file is part of the sos project: https://github.com/sosreport/sos
#
# This copyrighted material is made available to anyone wishing to use,
# modify, copy, or redistribute it subject to the terms and conditions of
# version 2 of the GNU General Public License.
#
# See the LICENSE file in the source distribution for further information.
```

This should appear beneath your own copyright declaration.